modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/CSPComplianceCheck.java

```diff
@@ -165,19 +170,42 @@ private void _checkIllegalTags(
 				if (fileName.endsWith(".jsp") || fileName.endsWith(".jspf") ||
 					fileName.endsWith(".jspx")) {
 
-					addMessage(
-						fileName,
-						StringBundler.concat(
-							"Use <aui:", tagName, "> tag instead of <", tagName,
-							">, see LPD-18227"),
-						lineNumber);
+					String substring = content.substring(lastIndex + 1, x);
```

Comment:
Inline `substring`, it's only used once and does not take up too much room.

Reply:
@drewbrokke fixed